components/file_attachment_list/file_attachment_list.test.tsx

```diff
@@ -4,17 +4,57 @@
 import {shallow} from 'enzyme';
 import React from 'react';
 
+import {Post} from 'mattermost-redux/src/types/posts';
+
 import FileAttachment from 'components/file_attachment';
 import SingleImageView from 'components/single_image_view';
 
 import FileAttachmentList from './file_attachment_list';
 
 describe('FileAttachmentList', () => {
-    const post = {id: 'post_id', file_ids: ['file_id_1', 'file_id_2', 'file_id_3']};
+    const post: Post = {
```

Comment:
For future reference, we have a `getPostMock` function in `utils/test_helper` that mocks out all the extra data you need here.

Reply:
Handy - thanks!